### PR TITLE
new icon: kubeflow (original, original-wordmark, plain, plain-wordmark, line, line-wordmark)

### LINKED ISSUE #2086: [ICON REQUEST]: Kubeflow
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Kubeflow is an open-source platform for machine learning and MLOps on Kubernetes introduced by Google.

### Links and sources

Official Website: https://www.kubeflow.org/

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by weh**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #2086 **

### CODE REVIEW

**Review by canaleal — CHANGES_REQUESTED** [review]
Nice! Thank you for the contribution!  😄 

The json looks excellent, but the SVGs need a few modifications.

The viewboxes for all SVGs are not `0 0 128 128` and the plain-wordmark svg has strokes that should be converted into a unified path.

**Comment by weh** [comment]
sorry @canaleal, looks like i committed the unfinished files. should be fixed now.

File: icons/kubeflow/kubeflow-line-wordmark.svg
Comment (canaleal) [inline icons/kubeflow/kubeflow-line-wordmark.svg:1]: The SVG is not centered, the tip of the triangle is cut off at the top. 
There's also a little bit of padding left over for the width and height.

image

File: icons/kubeflow/kubeflow-plain-wordmark.svg
Comment (canaleal) [inline icons/kubeflow/kubeflow-plain-wordmark.svg:1]: There's still a little bit of padding on this SVG and the `plain.svg`. 
It's honestly not much, but best to remove it. 

image

File: icons/kubeflow/kubeflow-line.svg
Comment (canaleal) [inline icons/kubeflow/kubeflow-line.svg:1]: The SVG has a width that's slightly over 128, causing the right side to be outside the viewbox. 

image

**Review by canaleal — DISMISSED** [review]
Great work. 

I left a few comments about the padding/alignment on some of the SVGs, but aside from that everything else looks excellent.

**Comment by weh** [comment]
@canaleal i fixed the image box issues. hope this time i got it right :crossed_fingers:

File: icons/kubeflow/kubeflow-line.svg
Comment (weh) [reply icons/kubeflow/kubeflow-line.svg:1]: fixed in https://github.com/devicons/devicon/pull/2117/commits/452be760b339f2625ff0ed74325145731977c8d0

File: icons/kubeflow/kubeflow-plain-wordmark.svg
Comment (weh) [reply icons/kubeflow/kubeflow-plain-wordmark.svg:1]: fixed in https://github.com/devicons/devicon/pull/2117/commits/452be760b339f2625ff0ed74325145731977c8d0

File: icons/kubeflow/kubeflow-line-wordmark.svg
Comment (weh) [reply icons/kubeflow/kubeflow-line-wordmark.svg:1]: fixed in https://github.com/devicons/devicon/pull/2117/commits/452be760b339f2625ff0ed74325145731977c8d0

**Review by canaleal — APPROVED** [review]

**Pull request merged**